deploy/installer/generated/ibm-block-csi-operator.yaml

```diff
@@ -1528,6 +1531,13 @@ rules:
   - list
   - watch
   - update
+- apiGroups:
+  - ""
+  resources:
+  - persistentvolumeclaims/status
+  verbs:
+  - update
+  - patch
```

Comment:
Why we need this ability in the operator?

Reply:
[for resizer sidecar](https://github.com/kubernetes-csi/external-resizer/blob/v1.0.0/deploy/kubernetes/rbac.yaml#L41)

Reply:
Thanks